docker: latest dev image missing picolibc.h?

The CI build for the TinyGo Drivers repo is currently failing. It appears that there is something missing from the Docker image related to the recent PR merged related to this.
 
```
tinygo build -size short -o ./build/test.hex -target=itsybitsy-m0 ./examples/adt7410/main.go
In file included from /tinygo/lib/picolibc-stdio.c:4:
In file included from /tinygo/lib/picolibc/newlib/libc/tinystdio/stdio.h:44:
In file included from /usr/lib/llvm-11/lib/clang/11.1.0/include/inttypes.h:21:
In file included from /tinygo/lib/picolibc/newlib/libc/include/inttypes.h:16:
/tinygo/lib/picolibc/newlib/libc/include/newlib.h:35:10: fatal error: 'picolibc.h' file not found
#include <picolibc.h>
         ^~~~~~~~~~~~
```
See https://app.circleci.com/pipelines/github/tinygo-org/drivers/988/workflows/46cbb77e-c7e0-4437-8ef7-98c45dd3da6e/jobs/1014?invite=true#step-105-8

--- a/Dockerfile
+++ b/Dockerfile
@@ -16,6 +16,7 @@ RUN cd /tinygo/ && \
     git submodule update --init --recursive --force
 
 COPY ./lib/picolibc-* /tinygo/lib/
+COPY ./lib/picolibc-include/* /tinygo/lib/picolibc-include/
 
 RUN cd /tinygo/ && \
     go install /tinygo/

